<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MrqSubUart</class>
 <widget class="QWidget" name="MrqSubUart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Frame</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="5" column="2">
       <widget class="QDoubleSpinBox" name="spinInvterval">
        <property name="suffix">
         <string>s</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="spinFrameLen"/>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="spinReceiveLen"/>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkSensor">
        <property name="text">
         <string>ON/OFF</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spinSof"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Length</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>SOF</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Interval</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Receive Num</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="spinBox">
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinSof</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinSof</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
